app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -30,35 +31,75 @@ AccountService = ($state, $auth) ->
       .then((response) ->
         if response.signedIn
           angular.copy(response, Service.loggedInUser)
+          Service._reformatDOB()
+          ShortFormApplicationService.importUserData(Service.loggedInUser)
       ).catch((response) ->
         alert("Error: #{response.errors[0]}")
       )
 
-  Service._formatDOB = ->
-    month = Service.userAuth.dob_month
-    day = Service.userAuth.dob_day
-    year = Service.userAuth.dob_year
-    formattedDOB = year + '-' + month + '-' + day
-    Service.userAuth.DOB = formattedDOB
+  Service.signOut = ->
+    $auth.signOut()
+      .then((response) ->
+        angular.copy({}, Service.loggedInUser)
+      )
 
-  # this runs on init of the app to check if we're logged upon arrival
+  # this gets run on init of the app in AngularConfig to check if we're logged in
   Service.validateUser = ->
     $auth.validateUser().then((response) ->
       # will only reach this state if user is logged in w/ a token
       angular.copy(response, Service.loggedInUser)
+      Service._reformatDOB()
+      ShortFormApplicationService.importUserData(Service.loggedInUser)
     )
 
   Service.loggedIn = ->
     !_.isEmpty(Service.loggedInUser) && Service.loggedInUser.signedIn
 
+  #################### helper functions
+  Service._formatDOB = ->
+    month = Service.userAuth.dob_month
+    day = Service.userAuth.dob_day
+    year = Service.userAuth.dob_year
+    formattedDOB = year + '-' + month + '-' + day
+    Service.userAuth.DOB = formattedDOB
+
+  # reverse of the above function
+  Service._reformatDOB = ->
+    return false if _.isEmpty(Service.loggedInUser)
+    split = Service.loggedInUser.DOB.split('-')
+    Service.loggedInUser.dob_year = parseInt(split[0])
+    Service.loggedInUser.dob_month = parseInt(split[1])
+    Service.loggedInUser.dob_day = parseInt(split[2])
+
```

Comment:
@chrisdolendo ^^ speaking of formatting/re-formatting, massaging/re-massaging... I wonder if all of those functions could get pulled out into their own `DataMassagingService` (or whatever) because it's useful both in the `AccountService` and `ShortFormApplicationService` and I sense the opportunity for some DRY-ness.